service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -42,310 +30,105 @@
-                    .organisation(respondentTwoOrganisation)
-                    .orgPolicyCaseAssignedRole(SolicitorRole.SOLICITORB.getCaseRoleLabel())
-                    .build())
-                .respondentPolicy2(buildOrganisationPolicy(SolicitorRole.SOLICITORC))
-                .respondentPolicy3(buildOrganisationPolicy(SolicitorRole.SOLICITORD))
-                .respondentPolicy4(buildOrganisationPolicy(SolicitorRole.SOLICITORE))
-                .respondentPolicy5(buildOrganisationPolicy(SolicitorRole.SOLICITORF))
-                .respondentPolicy6(buildOrganisationPolicy(SolicitorRole.SOLICITORG))
-                .respondentPolicy7(buildOrganisationPolicy(SolicitorRole.SOLICITORH))
-                .respondentPolicy8(buildOrganisationPolicy(SolicitorRole.SOLICITORI))
-                .respondentPolicy9(buildOrganisationPolicy(SolicitorRole.SOLICITORJ))
-                .build();
-
-            assertThat(extractedCaseData.getNoticeOfChangeAnswersData()).isEqualTo(expectedNoticeOfChangeAnswers);
-            assertThat(extractedCaseData.getRespondentPolicyData()).isEqualTo(expectedRespondentPolicyData);
+            assertThat(extractedAdditionalApplicationsBundle).isEqualTo(expectedAdditionalApplicationsBundle);
         }
 
         @Test
-        void shouldGenerateRespondentPoliciesAndNoticeOfChangeAnswersFromPartiallyPopulatedCaseData() {
-            List<Element<Applicant>> applicants = List.of(
-                element(Applicant.builder()
-                    .party(ApplicantParty.builder()
-                        .build())
-                    .build()));
+        void shouldNotRemoveSupportingEvidenceBundles() {
+            AdditionalApplicationsBundle additionalApplicationsBundle = buildExpectedAdditionalApplicationsBundle();
 
-            List<Element<Respondent>> respondents = List.of(
-                element(Respondent.builder()
-                    .party(RespondentParty.builder()
-                        .lastName("Simpson")
-                        .build())
-                    .build()),
-                element(Respondent.builder()
-                    .solicitor(RespondentSolicitor.builder()
-                        .organisation(Organisation.builder().build())
-                        .build())
-                    .party(RespondentParty.builder()
-                        .firstName("Mark")
-                        .build())
-                    .build()),
-                element(Respondent.builder()
-                    .solicitor(RespondentSolicitor.builder()
-                        .unregisteredOrganisation(UnregisteredOrganisation.builder()
-                            .name("Some address")
-                            .build())
-                        .build())
-                    .party(RespondentParty.builder()
-                        .build())
-                    .build()));
+            AdditionalApplicationsBundle expectedAdditionalApplicationsBundle =
```

Comment:
this can be removed. you can assert with the above variable `additionalApplicationsBundle`